README.md

```diff
@@ -113,6 +113,9 @@ Import Mandrill blacklist that you get from [here](https://mandrill.zendesk.com/
 
 If the list was successfully imported the CLI will return `OK`.
 
+### Import SendGrid Suppressions
+
+Follow [SendGrid Suppressions import guide](Usage_Sendgrid_Suppressions.md).
```

Comment:
I usually package the `Usage.md` with the release package. Should I include `Usage_Sendgrid_Suppressions.md`?

Should each of the commands be broken out into their own file like this one?